WickedEngine/wiScene_BindLua.cpp

```diff
@@ -3814,6 +3826,36 @@ int AnimationComponent_BindLua::SetEnd(lua_State* L)
 	return 0;
 }
 
+int AnimationComponent_BindLua::IsRootMotion(lua_State* L)
+{
+	wi::lua::SSetBool(L, component->IsRootMotion());
+	return 1;
+}
+
+int AnimationComponent_BindLua::RootMotionOn(lua_State* L)
+{
+	component->RootMotionOn();
+	return 0;
+}
+
+int AnimationComponent_BindLua::RootMotionOff(lua_State* L)
+{
+	component->RootMotionOff();
+	return 0;
+}
+
+int AnimationComponent_BindLua::GetRootTranslation(lua_State* L)
+{
+	wi::lua::SSetFloat3(L, component->rootTranslationOffset);
```

Comment:
These should use Vector lua binding instead of pushing a number of floats. 
```
Luna<Vector_BindLua>::push(L, component->rootTranslationOffset);
```